tests/async/test_browsercontext_add_cookies.py

```diff
@@ -372,21 +370,5 @@ async def test_should_not_block_third_party_cookies(
     )
     await page.frames[1].evaluate("document.cookie = 'username=John Doe'")
     await page.wait_for_timeout(2000)
-    allows_third_party = is_firefox
     cookies = await context.cookies(server.CROSS_PROCESS_PREFIX + "/grid.html")
-
-    if allows_third_party:
-        assert cookies == [
-            {
-                "domain": "127.0.0.1",
-                "expires": -1,
-                "httpOnly": False,
-                "name": "username",
-                "path": "/",
-                "sameSite": "Lax" if is_chromium else "None",
-                "secure": False,
-                "value": "John Doe",
-            }
-        ]
-    else:
-        assert cookies == []
+    assert cookies == []
```

Comment:
Why this change? Was there a similar one upstream?

Reply:
Yes see the linked upstream backport.